Tests/UnitTests/Misc/SandboxEnvironmentDetectorTests.swift

```diff
@@ -30,14 +30,6 @@ class SandboxEnvironmentDetectorTests: TestCase {
         expect(SystemInfo.with(receiptResult: .nilURL, inSimulator: false).isSandbox) == false
     }
 
-    func testMacSandboxReceiptIsSandbox() {
-        expect(SystemInfo.with(receiptResult: .macOSSandboxReceipt, inSimulator: false).isSandbox) == true
-    }
-
-    func testMacAppStoreReceiptIsNotSandbox() {
-        expect(SystemInfo.with(receiptResult: .macOSAppStoreReceipt, inSimulator: false).isSandbox) == false
-    }
```

Comment:
Removed because 1. we dont currently run tests on mac targets 2. the new way of testing for sandbox environment on macos does not depend on the receipt path